lib/service/storage/elasticsearch.js

```diff
@@ -210,6 +210,54 @@ class ElasticSearch extends Service {
       .catch(error => this._esWrapper.reject(error));
   }
 
+  /**
+   * Returns detailed multi-level storage stats data
+   *
+   * @returns {Promise.<Object>}
+   */
+  async stats () {
+    const esRequest = {
+      metric: ['docs', 'store'],
+    };
+
+    const { body } = await this._client.indices.stats(esRequest);
+
+    const response = {
```

Comment:
Nitpicking:
Why declaring a `response` object instead of directly declare `indexes` and `size_in_bytes` ?

Reply:
To prevent confusion and improve on readability of the logic (parsing) below. But it is not necessary indeed.